src/elements/content-sharing/stories/ContentSharing.stories.js

```diff
@@ -16,7 +16,8 @@ export const withModal = () => {
         <>
             <p>
                 Update the values in the Knobs section below to view the ContentSharing UI Element. The internal
-                SharingModal will appear when valid values have been entered.
+                SharingModal will appear when valid values have been entered. Make sure to add the current origin to to
+                to your app&apos;s list of CORS domains in the developer console.
```

Comment:
```suggestion
                SharingModal will appear when valid values have been entered. Make sure to add the current origin to 
                your app&apos;s list of CORS domains in the developer console.
```

Reply:
that's a lot of to's!

Reply:
yes it is :facepalm: i read it twice, _too_!